cime_config/config_grids.xml

```diff
@@ -1899,6 +1899,16 @@
       <mask>IcoswISC30E3r5</mask>
     </model_grid>
 
+    <model_grid alias="TL319_oQU240wLI_gis20" compset="_MALI">
+      <grid name="atm">TL319</grid>
+      <grid name="lnd">TL319</grid>
+      <grid name="ocnice">oQU240wLI</grid>
+      <grid name="rof">JRA025</grid>
```

Comment:
How is this different from the r025 mesh used in high-res cases?

Reply:
It's the mesh that the runoff for the JRA data forcing is on, but I don't know how it's different than our standard r025.  @jonbob , do you know?

It looks like there are about 29 instances of this used for existing JRA grids.